src/routes/upload/version/[id]/+page.svelte

```diff
@@ -353,7 +354,7 @@
         <input type="file" onchange={(e => {
           const target = e.target as HTMLInputElement;
           if (target && target.files && target.files.length == 1) {
-            if (target.files[0].size > 75 * 1024 * 1024) {
+            if (target.files[0].size > 150 * 1024 * 1024) {
```

Comment:
Would it be a good idea to consider pulling this hardcoded number into a configuration setting of some kind? I imagine there will be some discussion regarding the maximum file size for an uploaded mod, and it'll definitely be easier to change the config file than changing this magic number every time.

Reply:
Probably but the frontend doesn't have much configuration other than api url